components/CriteriaMitra/index.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+  <Portal to="criteriamitra">
+    <div
+      :class="{
+        'modal-criteria-mitra__backdrop': true,
+        'modal-criteria-mitra__backdrop--show': show,
+        'modal-criteria-mitra__backdrop--hide': !show,
+      }"
+      @click.self="onClose('overlay')"
+    >
+      <div
+        :class="{
+          'modal-criteria-mitra__wrapper': true,
+          'modal-criteria-mitra__wrapper--show': show,
+          'modal-criteria-mitra__wrapper--hide': !show,
+        }"
+      >
+        <div class="modal-criteria-mitra__close">
+          <BaseButton variant="circle" @click="onClose">
+            <template #icon>
+              <jds-icon class="text-white" size="20px" name="times" />
+            </template>
+          </BaseButton>
+        </div>
+        <div class="modal-criteria-mitra__container">
+          <div class="modal-criteria-mitra__container-heading">
+            Kriteria Mitra Yang Dapat Mengikuti
+          </div>
+          <div class="modal-criteria-mitra__container-content">
+            <div
+              v-for="item in criteriaMitraDetails"
+              :key="item.id"
+              class="modal-criteria-mitra__container-content-main"
+            >
+              <div class="modal-criteria-mitra__container-content-main-img">
+                <img class="mx-auto" width="40px" height="40px" :src="item.image" :alt="item.textImg">
```

Comment:
minor:

hello @doohanas I think we doesn't have to add `px` to `width` and `height` attribute on an <img> tag, cmiiw

Reply:
😆 You're right the `width `and `height` attrs doesn't need `unit`